viewer/templates/viewer/help.html

```diff
@@ -6,17 +6,23 @@
   </a>
 </nav>
 
-<h2 class="o-header_alt">Common web page index searches</h2>
+<h2 class="o-header_alt">Common Searches</h2>
 
 <p>
-  The web page index stores a nightly snapshot of webpage data that you can
-  search for all sorts of useful things. You can search for single terms or
-  exact phrases.
+  This tool stores a nightly snapshot of consumerfinance.gov web page data that
+  you can search for all sorts of useful things. Common search examples are
+  below. You can also
+  <a href="">learn more about the data</a>. View results in the browser, or
```

Comment:
Per the design, shouldn't there be a line break before "View results in the browser"? Also there's a missing href here.